drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/pages/AttributeColumnPage.java

```diff
@@ -78,7 +77,6 @@ public void prepareView() {
     private List<String> rawAttributes() {
         return new ArrayList<String>() {{
             addAll(attributesWithoutChooseAttribute());
-            add(GuidedDecisionTable52.NEGATE_RULE_ATTR);
```

Comment:
@jomarko Just curious Why this is no more needed?

Reply:
@yesamer it is contained now in `attributesWithoutChooseAttribute()`